ci-operator/step-registry/hypershift/aws/ephemeral/hypershift-aws-ephemeral-workflow.yaml

```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: hypershift-aws-ephemeral
+  documentation: |-
+    The HyperShift aws ephemeral workflow creates a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in the `osd-hypershift` AWS account and is destroyed after tests
```

Comment:
No, right now this must be configured with a `cluster_profile` that sets up a standalone cluster or it will not work, as we don't set a kubeconfig anywhere in these steps.

Compare:
* https://github.com/openshift/release/blob/4c2e8e270f34bf49860b9f68ce3d977a12809581/ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml (what you are using)
* https://github.com/openshift/release/blob/4c2e8e270f34bf49860b9f68ce3d977a12809581/ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml#L11 what we use in CI to get a mgmt cluster and where we explicitly mount and set the kubeconfig

Reply:
Agreed.  Cluster-bot should only interact with a cluster that already has hypershift installed and only ever return the HC's connection information to the user.